lib/core/src/model.rs

```diff
@@ -545,6 +547,9 @@ pub struct PaymentSwapData {
 #[derive(Debug, Clone, PartialEq, Serialize)]
 pub struct Payment {
     /// The tx ID of the onchain transaction
+    ///
+    /// For Receive Payments that are [PaymentState::Pending], this may be the lockup or the claim
+    /// tx ID. For Receive Payments that are [PaymentState::Complete], this is the claim tx ID.
```

Comment:
I don't think the lockup in case of receive should be the tx_id.
It seems to me that tx_id should only identify a transaction of the user's onchain wallet.